pipelines/commands/syncstatus_test.go

```diff
@@ -0,0 +1,78 @@
+package commands
+
+import (
+	"bytes"
+	"github.com/jfrog/build-info-go/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles/state"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/tests"
+	"github.com/jfrog/jfrog-client-go/pipelines/services"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"time"
+)
+
+func initDisplaySyncStatusTest(t *testing.T) (*bytes.Buffer, func()) {
```

Comment:
Notice that the code of this function duplicated twice as part of this PR.
Let's create a file named `utils_test.go` inside this package, move this function to it, rename it appropriately, and reuse it.